### PR TITLE
Fixes #7418 - fixed capitalization for some AR errors

### PR DESCRIPTION
**Pull request by lzap**
Cosmetic thingy.

### CODE REVIEW

File: app/models/concerns/orchestration/dhcp.rb [hunk 1]
```diff
@@ -124,7 +124,7 @@ def ip_belongs_to_subnet?
     return if subnet.nil? or ip.nil?
     return unless dhcp?
     unless subnet.contains? ip
-      errors.add(:ip, _("Does not match selected Subnet"))
+      errors.add(:ip, _("does not match selected Subnet"))
```
Comment (domcleal) [inline app/models/concerns/orchestration/dhcp.rb]: lower case 'Subnet'

File: app/models/user.rb [hunk 2]
```diff
@@ -466,14 +466,14 @@ def ensure_privileges_not_escalated
...
-      errors.add :role_ids, _("You can't assign some of roles you selected")
+      errors.add :role_ids, _("you can't assign some of roles you selected")
     end
   end
 
   def ensure_admin_not_escalated
     admin_check = self.new_record? ? self.admin? : self.admin_changed?
     if admin_check && !User.current.can_change_admin_flag?
-      errors.add :admin, _("You can't change Administrator flag")
+      errors.add :admin, _("you can't change Administrator flag")
```
Comment (domcleal) [inline app/models/user.rb]: lower 'A'?

**Comment by isratrade** [comment]
@lzap, I agree with @domcleal's comments.  After updated, I think it's ready to merge.

**Comment by lzap** [comment]
Both fixed.

**Comment by isratrade** [comment]
Merged as fb69591a46c1f11dda4926ccb7ec23b744307192. Thanks @lzap !

**Pull request closed without merging**